angr/storage/memory_mixins/paged_memory/pages/mv_list_page.py

```diff
@@ -26,19 +27,21 @@ class MVListPage(
     def __init__(self, memory=None, content=None, sinkhole=None, mo_cmp=None, **kwargs):
         super().__init__(**kwargs)
 
-        self.content: List[Optional[Union[_MOTYPE,Set[_MOTYPE]]]] = content
+        self.content: DynamicDictList[Optional[Union[_MOTYPE,Set[_MOTYPE]]]] = \
+            DynamicDictList(max_size=memory.page_size, content=content) if content is not None else None
         self.stored_offset = set()
         self._mo_cmp: Optional[Callable] = mo_cmp
 
-        if content is None:
+        if self.content is None:
             if memory is not None:
-                self.content: List[Optional[Union[_MOTYPE,Set[_MOTYPE]]]] = [None] * memory.page_size
+                self.content: DynamicDictList[Optional[Union[_MOTYPE,Set[_MOTYPE]]]] = \
```

Comment:
How does this type annotation work? What did you do to get the class to have type parameters like this?

Reply:
I see. this is one of the TODOs. never mind!

Reply:
Fixed!